scripts/install.py

```diff
@@ -69,40 +72,45 @@
 # for use in `download_from_pypi`
 from distutils.version import LooseVersion
 
-from fnmatch import fnmatch
-
 import os
 from os import path
 
 import re
 
 import shutil
 
-from subprocess import call, check_call, check_output, CalledProcessError
+from subprocess import check_call, check_output, CalledProcessError
 
 from urllib2 import urlopen
 
+locale.setlocale(locale.LC_ALL, 'C')
 
 ## defaults and constants
 
-PROG="GC3Pie install"
+PROG = "GC3Pie install"
+
+VIRTUALENV_LATEST_URL = "https://raw.github.com/pypa/virtualenv/master/virtualenv.py"
+VIRTUALENV_191_URL = "https://raw.github.com/pypa/virtualenv/1.9.1/virtualenv.py"
 
-VIRTUALENV_LATEST_URL="https://raw.github.com/pypa/virtualenv/master/virtualenv.py"
-VIRTUALENV_191_URL="https://raw.github.com/pypa/virtualenv/1.9.1/virtualenv.py"
 
 class default:
-    BASE_PIP_URL="https://pypi.python.org/simple"
-    GC3PIE_REPO_URL="https://github.com/uzh/gc3pie.git"
+    BASE_PIP_URL = "https://pypi.python.org/simple"
+    GC3PIE_REPO_URL = "https://github.com/uzh/gc3pie.git"
     TARGET = path.expandvars('$HOME/gc3pie')
     UNRELEASED = False
     WITH_APPS = True
+    ANACONDA = False
```

Comment:
I do not much like using `default.ANACONDA` as a test of whether the we're installing with Anaconda Python: for one, it's not a default (it's a user choice / environmental accident we have no control over).  Why not rename it to a global variable like e.g. `python_is_anaconda` or `using_anaconda` or `running_anaconda` or similar?